src/equations/compressible_euler_2d.jl

```diff
@@ -2018,8 +2018,18 @@ end
     return cons2entropy(u, equations)
 end
 
-# Calculate the modified specific entropy of Guermond et al. (2019): s_0 = p * rho^(-gamma) / (gamma-1).
-# Note: This is *not* the "conventional" specific entropy s = ln(p / rho^(gamma)).
+@doc raw"""
```

Comment:
Why the `@doc raw"""...` style instead of just using `"""`?

Reply:
With `"""` the latex code did not work immediately. I think, then, I would have to use two `\` every time. Is there a disadvantage of `@doc raw"""`. If so, I can try the other thing.

Reply:
All good, I was just curious